app/config/AppConfig.scala

```diff
@@ -71,7 +71,14 @@ class AppConfig @Inject()(val config: Configuration, servicesConfig: ServicesCon
 
   lazy val whatYouOweUrl: String = config.get[String]("urls.whatYouOwe")
 
-  lazy val feedbackUrl: String = config.get[String]("urls.feedbackUrl")
+  lazy val contactFrontendHost: String = s"${servicesConfig.baseUrl("contact-frontend")}/"
+  val contactFrontendStartUrl: String = "feedback.url"
+  val contactFrontendServiceId: String = "contact-frontend.serviceId"
```

Comment:
Can we do the config.get's here? so we are just combining strings in the feedbackUrl val